astropy/units/tests/test_quantity_array_methods.py

```diff
@@ -511,6 +511,7 @@ def test_not_implemented(self):
         with pytest.raises(NotImplementedError):
             q1.tolist()
         with pytest.raises(NotImplementedError):
+            # FIXME: tobytes instead ?
             q1.tostring()
```

Comment:
@mhvk - What to do about this one? Should `tobytes` be added here and in `astropy/units/quantity.py` ?

Reply:
Yes, I think it was an oversight not to have those. If you could add those, that would be great!

Reply:
@mhvk - Ok, done.